admin/autoupdater/src/autoupdater.c

```diff
@@ -56,8 +56,10 @@
 static const char *const download_d_dir = "/usr/lib/autoupdater/download.d";
 static const char *const abort_d_dir = "/usr/lib/autoupdater/abort.d";
 static const char *const upgrade_d_dir = "/usr/lib/autoupdater/upgrade.d";
+static const char *const check_d_dir = "/usr/lib/autoupdater/check.d";
 static const char *const lockfile = "/var/lock/autoupdater.lock";
 static const char *const firmware_path = "/tmp/firmware.bin";
+static const char *const info_path = "/tmp/info.txt";
```

Comment:
i think a more verbose name would be good, like something that includes the package name and something about the content, maybe something like "autoupdater-fw-update-info.txt" ?